code/datums/gamemode/factions/bloodcult/bloodcult_projectiles.dm

```diff
@@ -286,5 +305,76 @@
 			return 0
 	return 1
 
+
+/obj/item/projectile/blooddagger/bump_original_check()
+	if(!bumped)
+		if(loc == get_turf(original))
+			if(!(original in permutated))
+				to_bump(original)
+
 /obj/item/projectile/blooddagger/cultify()
 	return
+
+
+/obj/effect/overlay/bloodnail
+	name = "blood nail"
+	desc = "A pointy red nail, appearing to pierce not through what it rests upon, but through the fabric of reality itself."
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "bloodnail"
+	anchored = 1
+	density = 0
+	plane = ABOVE_HUMAN_PLANE
+	layer = CLOSED_CURTAIN_LAYER
+	var/atom/stuck_to = null
+	var/duration = 100
+
+/obj/effect/overlay/bloodnail/New()
+	..()
+	pixel_x = rand(-4, 4) * PIXEL_MULTIPLIER
+	pixel_y = rand(-4, 4) * PIXEL_MULTIPLIER
+
+/obj/effect/overlay/bloodnail/Destroy()
+	if(stuck_to)
+		unlock_atom(stuck_to)
+	stuck_to = null
+	..()
+
+/obj/effect/overlay/bloodnail/cultify()
+	return
+
+/obj/effect/overlay/bloodnail/proc/stick_to(var/atom/A, var/side = null)
+	pixel_x = rand(-4, 4) * PIXEL_MULTIPLIER
+	pixel_y = rand(-4, 4) * PIXEL_MULTIPLIER
+	playsound(A, 'sound/items/metal_impact.ogg', 30, 1)
+	var/turf/T = get_turf(A)
+	loc = T
+	playsound(T, 'sound/weapons/hivehand_empty.ogg', 75, 1)
+
+	if(isturf(A))
```

Comment:
`if (A == T)` is clearer and has the same logic. Add a comment explaining it for good measure.

Reply:
Maybe the nails could stick to floors when fired by the wraith but to_bump() will now only let it stick to living mobs, so I'll just remove that segment altogether.